### ISSUE TITLE
\paragraph{} sectioning go out of sync when using the scrbook class

### ISSUE THREAD

**Issue by mxordn** [maieul/ledmac#331]
In a layout with parallel pages the paragraphs where the \paragraph{} sectioning command is used, go out of sync. The issue only appears useing the KOMA classes:

    \documentclass[11pt,a4paper,BCOR=12mm,DIV=14]{scrbook}
    \usepackage[utf8]{inputenc}
    \usepackage[ngerman, polutonikogreek, latin]{babel}
    \usepackage[T1]{fontenc}
    \usepackage{eledmac}
    \usepackage{eledpar}
    
    
    \begin{document}
    \begin{pages}
    \begin{Leftside}
    \selectlanguage{latin}
    \beginnumbering
    \pstart
    \paragraph{bla} phantomtext phantomtextphantomtext phantomtext phantomtext phantomtext
    \pend\pstart\itshape
    \paragraph{2.} phantomtext2 phantomtext2 phantomtext2 phantomtext2 phantomtext2 phantomtext2
    \pend\pstart\itshape
    \paragraph{3.} phantomtext3 phantomtext3 phantomtext3 phantomtext3 phantomtext3 phantomtext3 phantomtext3.
    \pend
    \endnumbering
    \end{Leftside}
    %ENDE LINKE SEITE
    
    
    %BEGINN RECHTE SEITE
    \begin{Rightside}
    \selectlanguage{ngerman}
    \beginnumbering
    \pstart
    \paragraph{bla} phantomtext phantomtextphantomtext phantomtext phantomtext phantomtext phantomtext
    \pend\pstart\itshape
    \paragraph{2. bla} phantomtext2 phantomtext2 phantomtext2 phantomtext2 phantomtext2 phantomtext2
    \pend\pstart\itshape
    \paragraph{3.} phantomtext3 phantomtext3 phantomtext3 phantomtext3 phantomtext3.
    \pend
    \endnumbering
    \end{Rightside}
    \end{pages}
    \Pages
    \end{document}

**Comment by floriandk** [comment]
This is not a bug -- you need to use the sectioning commands differently, cf. chapter 14 (eledmac vers. 1.24) of the handbook.

* Don't use LaTeX's standard sectioning-commands inside numbered text, use the special variants starting with `\eled...`e.g. `\eledsection`
* These have to stand in a `\pstart` - `\pend` for themselves; the following text goes to an extra one.

I don't know why, but there is no `\eledparagraph`-command (yet). But you could always imitate its behaviour by something like `\textbf{paragraph heading} \quad your text`.

**Comment by mxordn** [comment]
Thanks Florian, but that's exaclty the problem with the \paragraph command. I think it can't easily be "translated" into \eledparagraph{}, because of its behaviour that the text runs on in the same line after the marked up title. With a `\pstart […] \pend` this would cause a linebreak in parallel layout.

My workaround for now would be to build it "manually" with something similar to what you've suggested. If you're not in need of counting the paragraph-titles for any reason, then the command in preamble could look like:

`\renewcommand{\paragraph}[1]{\noindent\textbf{\sffamily #1}\quad}`

**Comment by floriandk** [comment]
You are probably right about the reason, why Maïeul didn't supply a `\eledparagraph` yet.

I'd suggest that you close this issue, as there obviously isn't a bug in the syncing. If you want to ask for the implementation of `\eledparagraph` (and perhaps `\eledsubparagraph`) it would be better, if you open another ticket with a feature request and a fitting title. This makes it easier for the maintainer to keep track.

**Comment by maieul** [comment]
I agree with Florian.

**Comment by mxordn** [comment]
Ok I'll close this issue. I think its a general problem of the compatibility of eledmac and KOMA-Script. Cf.  #337.
